core/src/main/kotlin/io/specmatic/stub/api.kt

```diff
@@ -477,3 +495,9 @@ fun isOpenAPI(path: String): Boolean =
         false
     }
 
+fun exitIfInvalidExamplesDirExists(exampleDirPaths: List<String>) {
```

Comment:
sounds very similar to `exitIfDoesNotExist`, dir or file, can we not reuse the same function?